platform/darwin/src/MGLStyle.h

```diff
@@ -318,6 +318,27 @@ MGL_EXPORT
  */
 - (void)removeSource:(MGLSource *)source;
```

Comment:
Are we going to live with both methods or should we deprecate this?

Reply:
I think we can hold off on deprecation for the moment.